packages/postgres-database/src/simcore_postgres_database/utils_products.py

```diff
@@ -30,3 +43,42 @@ async def get_default_product_name(conn: _DBConnection) -> str:
 
     assert isinstance(product_name, str)  # nosec
     return product_name
+
+
+async def get_product_group_id(
+    connection: _DBConnection, product_name: str
+) -> Optional[int]:
+    group_id = await connection.scalar(
+        sa.select([products.c.group_id]).where(products.c.name == product_name)
+    )
+    return None if group_id is None else int(group_id)
+
+
+async def get_or_create_product_group(
+    connection: _AiopgConnection, product_name: str
+) -> int:
```

Comment:
don't we have a GroupID alias somewhere? or is this not available in this package?